pkg/diagnose/connectivity/trace_request.go

```diff
@@ -112,10 +114,14 @@ func createEndpointURL(domain string, endpointInfo endpointInfo) string {
 	return domain + endpointInfo.Endpoint.Route
 }
 
+// vertifyEndpointResponse interprets the endpoint response and displays information on if the connectivity
+// check was successful or not
 func verifyEndpointResponse(writer io.Writer, statusCode int, responseBody []byte, err error) {
 
 	if err != nil {
 		fmt.Fprintf(writer, "could not get a response from the endpoint : %v\n", scrubber.ScrubLine(err.Error()))
+		noResponseHints(writer, err)
+		fmt.Fprintf(writer, "Could not get a valid response from the backend ====> %v\n", color.RedString("FAIL"))
```

Comment:
Looks pretty similar to the log on line 122, can these be combined? Just add a red `FAIL` to the first one maybe?

Reply:
Yeah you're right, thanks ! Done here f174de6cb111145836e762b5d64f2a8de38dcede